app/components/FilterableComponents/FilterableTablePagination.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {useRouter} from 'next/router';
+import {Pagination, Dropdown} from 'react-bootstrap';
+import safeJsonParse from 'lib/safeJsonParse';
+
+interface Props {
+  totalCount: number;
+}
+
+export const DEFAULT_MAX_RESULTS = 20;
+
+const FilterableTablePaginationComponent: React.FunctionComponent<Props> = (
+  props
+) => {
+  const {totalCount} = props;
+  const router = useRouter();
+
+  const pageData = router.query.pageVars
+    ? safeJsonParse(router.query.pageVars as string)
+    : {};
+
+  const maxResultsPerPage = pageData?.max_results || DEFAULT_MAX_RESULTS;
```

Comment:
```suggestion
  const maxResultsPerPage = pageData.max_results || DEFAULT_MAX_RESULTS;
```

don't need optional chaining here